### PR TITLE
history fixes

### PR DESCRIPTION
**Pull request by ginold**
- warning title in history page css fix
- radio buttons alignment
- 'version' variable added to outing history page
- correct date displayed in history table
- display the latest version of document from diff page

related to https://github.com/c2corg/v6_ui/issues/408 and  https://github.com/c2corg/v6_ui/issues/344

Check:
/waypoints/733992/en/1160836
/outings/diff/736707/fr/1164428/1164441
/outings/736707/fr/1164428
/routes/736728/fr/1164452

### CODE REVIEW

**Comment by ginold** [comment]
@arnaud-morvan, could you have a look on this PR plase?

**Comment by ginold** [comment]
@gberaudo you could also have a look at this after we merge the progressbar?

File: c2corg_ui/templates/helpers/view.html [hunk 1]
```diff
@@ -184,8 +184,7 @@ <h3 class="heading show-phone" ng-click="mainCtrl.animateHeaderIcon($event)" dat
             ng-if="userCtrl.auth.isAuthenticated()"></app-delete-association>
         % endif
       </div>
-    </div>
```
Comment (arnaud-morvan) [inline c2corg_ui/templates/helpers/view.html]: Are you sure you should remove this closing tag

**Comment by arnaud-morvan** [comment]
When I open an history page, I have the Plus flat button with Translate in other lang option. Is this normal ? When I click on it it seems to open an empty modal dialog.

**Comment by ginold** [comment]
I think you are right, there should be no buttons at all because you're just consulting an old version and should not be able to do anything with it.

**Comment by arnaud-morvan** [comment]
Missing update on locale files (.pot and .po)

**Comment by ginold** [comment]
Locale files update is not very important for the moment because it's always changing and in the end the translations will be made by an external enterprise.

**Comment by arnaud-morvan** [comment]
Yes, this is not very important, but this made my working dir dirty when I build your branch.

**Comment by ginold** [comment]
updated

**Comment by ginold** [comment]
sorry, not yet here!

**Comment by ginold** [comment]
If I remember well, Alex removed the locale compiling when running make check or serve because it was so annoying and we never needed those locales.

**Pull request merged**